okhttp-tests/src/test/java/com/squareup/okhttp/CacheTest.java

```diff
@@ -601,7 +600,7 @@ private void testClientPrematureDisconnect(TransferKind transferKind) throws IOE
     server.enqueue(new MockResponse()
         .setBody("B"));
 
-    URL url = server.getUrl("/?foo=bar");
+    HttpUrl url = server.url("/").newBuilder().addQueryParameter("foo", "bar").build();
```

Comment:
Note the change that had to be made here. The "path" parameter for this method is strictly path.